package/zoneinfo-utils/package

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+# Copyright (c) 2020 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+
+# Inspired by https://github.com/archlinux/svntogit-packages/blob/packages/tzdata/trunk/PKGBUILD
+
+pkgnames=(zoneinfo-utils)
+pkgdesc="Utilities for interacting with zoneinfo files"
+url=https://www.iana.org/time-zones
+_tzver=2020f
+pkgver="$_tzver"-1
+timestamp=2020-05-04T06:16Z
+section=utils
+maintainer="Eeems <eeems@eeems.email>"
+license="custom: public domain"
```

Comment:
This value of the license field conflicts with [the format required in the docs](https://github.com/toltec-dev/toltec/blob/stable/docs/package.md#license). I guess we should update the docs with some allowed exceptions such as this one.

Reply:
Opened #236